utils/check_docstrings.py

```diff
@@ -331,6 +331,7 @@
     "IBertModel",
     "IdeficsConfig",
     "IdeficsProcessor",
+    "IJepaModel",
```

Comment:
this is due to the modular model converter overwriting the `IJEPA_START_DOCSTRING` variable, leading to it not matching the function signature and when we do the repo consistency check, it fails, this happens similarly to the original ViT model, since it's also in this ignore list.

Reply:
I suppose with modular we can overwrite class/method docstring too, let me know if this is not the case